ui/admin/app/components/form/field/list-wrapper/key-value/index.hbs

```diff
@@ -14,13 +14,11 @@
 
   {{#let (unique-id) (unique-id) as |keyHeaderID valueHeaderID|}}
     <F.Control>
-      <Hds::Table class='key-value-list' name={{@name}}>
+      <Hds::Table class='list-wrapper' name={{@name}}>
```

Comment:
Should this class still be `list-wrapper` here?

Reply:
updated!